docs/authors.rst

```diff
@@ -0,0 +1 @@
+.. include:: ../AUTHORS.rst
```

Comment:
Should not be there in my opinion. Authors are long overhauled with code owners or no specific information at all.
https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners